packages/paste-website/src/components/open-graph-card/types.ts

```diff
@@ -0,0 +1,11 @@
+export type PackageData = {
+  name: string;
+  description: string;
+  Feature: string;
+  Documentation: boolean;
+  Figma: string;
+  Code: string;
+  Design_committee_review: string | null;
+  Engineer_committee_review: string | null;
```

Comment:
Why are these strings and not booleans?

Reply:
I inherited these types from prior work on component statuses, but it bothered me too. Looks like it's because they're coming from Airtable's API this way and we aren't remapping them anywhere.